### PR TITLE
Cal-340 Resolve SonarQube Identified Vulnerabilities

### PR DESCRIPTION
**Pull request by josephthweatt**
#### What does this PR do?
Sonarqube has identified potential vulnerabilities in Alliance which should be addressed before the next release. The issues can be found at https://sonarcloud.io/project/issues?id=org.codice%3Aalliance&resolved=false&types=VULNERABILITY

#### Who is reviewing it?
@vinamartin @mcalcote 
#### Choose 2 committers to review/merge the PR.
@clockard 
@shaundmorris 
#### What are the relevant tickets?

[CAL-340](https://codice.atlassian.net/browse/CAL-340)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliManagerType.java [hunk 1]
```diff
@@ -44,10 +44,6 @@ public String getSpecName() {
         return specName;
     }
 
-    public void setSpecName(String specName) {
-        this.specName = specName;
-    }
```
Comment (emmberk) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliManagerType.java]: ✏️ Can specName be `final` now that this method is removed?

Comment (josephthweatt) [reply catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliManagerType.java]: @emmberk  specName is being set in the constructor.

Comment (emmberk) [reply catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliManagerType.java]: Right. `specName` should only ever be set once in the constructor.

**Comment by josephthweatt** [comment]
build now

**Comment by josephthweatt** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by josephthweatt** [comment]
build now

**Comment by josephthweatt** [comment]
build now

**Comment by josephthweatt** [comment]
build now

**Pull request merged**